.github/workflows/black.yml

```diff
@@ -0,0 +1,32 @@
+# GitHub Action that uses Black to reformat the Python code in an incoming pull request.
+# If all Python code in the pull request is compliant with Black then this Action does nothing.
+# Othewrwise, Black is run and its changes are committed back to the incoming pull request.
+# https://github.com/cclauss/autoblack
+# Thanks to the PSF!
+# https://github.com/psf/black/actions/runs/17913292/workflow
+
+name: autoblack
+on: [pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Install Black
+        run: pip install black
+      - name: Run black --check .
+        run: black --check .
+      - name: If needed, commit black changes to the pull request
+        if: failure()
+        run: |
+          black .
+          git config --global user.name 'autoblack'
+          git config --global user.email 'cclauss@users.noreply.github.com'
```

Comment:
I guess we should change this email address?

Reply:
I thought I had already changed it to "geekzonebot@users.noreply.github.com" in [b306ddc](https://github.com/GeekZoneHQ/web/pull/184/commits/b306ddc562e882ba04c41403d18174d4e433ab5a)? Seems to be there now.